dozer-tests/src/e2e_tests/cases/eth/dozer-config.yaml

```diff
@@ -1,34 +1,35 @@
 app_name: working_app
 connections:
-- authentication: !Ethereum
-    filter:
-      from_block: 0
-      addresses: []
-      topics: []
-    wss_url: "{{ETH_WSS_URL}}"
+  - authentication: !Ethereum
+      filter:
+        from_block: 0
+        to_block: 1
```

Comment:
What does `to_block` 1 mean? Only fetch one block of data to save API cost?

Reply:
This test is basically fetching all historical data of ETH based on this query. by setting to_block, we are only bringing the first block. Firstly predictable test case, secondly we contain the data to a smaller data set.

Reply:
Every txn has a `block_no`
https://etherscan.io/block/16431869